Cargo.toml

```diff
@@ -49,34 +48,32 @@ git = "https://github.com/KZen-networks/class-groups"
 tag = "v0.1.4"
 optional = true
 
-[dependencies.rocket]
-version = "0.4.2"
-default-features = false
-optional = true
-
-
-
-[patch.crates-io]
-rust-gmp = { version = "0.5.0", features = ["serde_support"], git = "https://github.com/KZen-networks/rust-gmp" }
-
 [dev-dependencies]
-criterion = "0.2"
+criterion = "0.3"
+rust-crypto = "0.2"
 hex = "0.3.2"
-rocket_contrib = {version = "0.4.2",default-features = false,features = ["json"]}
-rocket = {version = "0.4.2", default-features = false}
-reqwest = {version = "0.9.5", default-features = false}
+rocket = "0.4.2"
+rocket_contrib = "0.4.2"
+reqwest = { version = "0.9", default-features = false }
```

Comment:
I see `request` now uses `"0.9"` instead of `"0.9.5"`. 
I believe it means that it will automatically updates of any version "0.9.X". 
I am for freezing versions unless there's a concrete reason to upgrade. Among other reasons - to prevent malicious code injection via dependency change.
Any particular reason to change here?

Reply:
dev-dependency is for testing, I see no reason to stick at a specific version